packages-presets/transformer-directives/src/screen.ts

```diff
@@ -24,10 +25,10 @@ export function handleScreen({ code, uno }: TransformerDirectivesContext, node:
   const resolveBreakpoints = () => {
     let breakpoints: Record<string, string> | undefined
     if (uno.userConfig && uno.userConfig.theme)
-      breakpoints = (uno.userConfig.theme as Theme).breakpoints
+      breakpoints = (uno.userConfig.theme as Theme).breakpoints || (uno.config.theme as Theme).breakpoints
```

Comment:
I don't think this is a safe way. I think you need to use the corresponding key name through the currently used preset